app/src/org/commcare/activities/DispatchActivity.java

```diff
@@ -63,7 +64,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
+        CrashUtil.init(this);
```

Comment:
What was the reason for moving this from `CommCareApplication` to here?

Reply:
Since this the only entry point for the application, I added it here.  It can go in `CommCareApplication` as well.

Reply:
Hm, I imagine it shouldn't go in 2 places, I just wasn't sure why it was necessary to move where this happens, since `CommCareApplication.onCreate()` will also always get called when the app starts up.